components/support/rc_crypto/nss/nss_sys/build.rs

```diff
@@ -56,7 +57,53 @@ fn env_str(name: &str) -> Option<String> {
     env::var(name).ok()
 }
 
+fn env_flag(name: &str) -> bool {
+    match env_str(name).as_ref().map(String::as_ref) {
+        Some("1") => true,
+        Some("0") => false,
+        Some(s) => {
+            println!(
```

Comment:
Will this prove to be annoying?

Reply:
Doubt it. If it's a flag what else would you set it to?